internal/k8s/handlers.go

```diff
@@ -523,6 +523,7 @@ func createAppProtectPolicyHandlers(lbc *LoadBalancerController) cache.ResourceE
 	return handlers
 }
 
+// compareSpecs returns true if the resources are different.
 func compareSpecs(oldresource, resource *unstructured.Unstructured) (bool, error) {
```

Comment:
Would it help to rename the function "areResourcesDifferent" ?

Reply:
good suggestion